### PR TITLE
chore: bump eslint-plugin-vue to v9

### PR DESCRIPTION
**Pull request by cexbrayat**
eslint-plugin-vue v9 has been released and yields warnings if not used with vue-eslint-parser v9 (which is the case if you naively bump the version in a freshly created `create-vue` project).

See https://github.com/vuejs/eslint-plugin-vue/issues/1893 for context

This PR bumps the version of both dependencies. Ideally, this should be released and used in new create-vue projects (I can open the following PRs when this one is merged and released)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,14 +41,14 @@
   },
   "peerDependencies": {
     "eslint": "^6.2.0 || ^7.0.0 || ^8.0.0",
-    "eslint-plugin-vue": "^8.0.1"
+    "eslint-plugin-vue": "9.0.0"
```
Comment (ota-meshi) [inline package.json:44]: Probably a typo.

```suggestion
    "eslint-plugin-vue": "^9.0.0"
```

**Comment by Fuzzyma** [comment]
So what is this PR waiting for?

**Comment by ivanlogiudice** [comment]
+1

**Comment by daegontaven** [comment]
Is there a reason this isn't merged yet?

**Comment by fx100** [comment]
Why haven't they merged yet?

**Comment by TheSeg** [comment]
Checking in on progress. This is preventing moving forward with now three releases of [vuejs/eslint-plugin-vue](https://github.com/vuejs/eslint-plugin-vue/releases)

**Comment by MartinX3** [comment]
I don't understand it.
There is the default `dependabot` and you can create a github actions CI to automerge patch and minor version dependency jumps...

**Comment by cexbrayat** [comment]
Please stop commenting on this issue: the maintainers of this repo will merge this as soon as they can. They all do this in their free time. You're just adding noise, and things won't go faster.

In the meantime, you _can_ upgrade to eslint-plugin-vue v9 if you really want to. You just have to install it manually and use a resolution to force `vue-eslint-parser` to v9 as well.

```
"resolutions": {
    "//": "remove vue-eslint-parser resolution when https://github.com/vuejs/eslint-config-typescript/pull/42 is merged",
    "vue-eslint-parser": "9.0.2"
  }
```

**Pull request merged**